simtools/util/model.py

```diff
@@ -83,3 +64,88 @@ def validateModelParameter(parNameIn, parValueIn):
             parType = MODEL_PARS[parNameModel]['type']
             return parNameModel, parType(parValueIn)
     return parNameIn, parValueIn
+
+
+def getCameraName(telescopeName):
+    '''
+    Get camera name from the telescope name.
+
+    Parameters
+    ----------
+    telescopeName: str
+        Telescope name (ex. South-LST-1)
+
+    Returns
+    -------
+    str
+        Camera name (validated by util.names)
+    '''
+    cameraName = ''
+    telSite, telClass, telType = names.splitTelescopeName(telescopeName)
+    if telClass == 'LST':
+        cameraName = 'LST'
+    elif telClass == 'MST':
+        if 'FlashCam' in telType:
+            cameraName = 'FlashCam'
+        elif 'NectarCam' in telType:
+            cameraName = 'NectarCam'
+        else:
+            logger.error('Camera not found for MST class telescope')
+    elif telClass == 'SCT':
+        cameraName = 'SCT'
+    elif telClass == 'SST':
+        if 'ASTRI' in telType:
+            cameraName = 'ASTRI'
+        elif 'GCT' in telType:
+            cameraName = 'GCT'
+        elif '1M' in telType:
+            cameraName = '1M'
+        else:
+            cameraName = 'SST'
+    else:
+        logger.error('Invalid telescope name - please validate it first')
+
+    cameraName = names.validateCameraName(cameraName)
+    logger.debug('Camera name - {}'.format(cameraName))
+    return cameraName
+
+
+def getTelescopeClass(telescopeName):
+    '''
+    Get telescope class from telescope name.
+
+    Parameters
+    ----------
+    telescopeName: str
+        Telescope name (ex. South-LST-1)
+
+    Returns
+    -------
+    str
+        Telescope class (SST, MST, ...)
+    '''
+    telSite, telClass, telType = names.splitTelescopeName(telescopeName)
+    return telClass
+
+
+def isTwoMirrorTelescope(telescopeName):
+    '''
+    Check if the telescope is a two mirror design.
+
+    Parameters
+    ----------
+    telescopeName: str
+        Telescope name (ex. South-LST-1)
+
+    Returns
+    -------
+    bool
+        True if the telescope is a two mirror one.
+    '''
+    telSite, telClass, telType = names.splitTelescopeName(telescopeName)
+    if telClass == 'SST':
+        return False if '1M' in telType else True
```

Comment:
Well, if we agree that it's not possible to validate and convert the telType part of the name, we can only use one option for each case, which has to be the same one as the one in the DB. This function to convert the names is not the only place we need to use the telescope names.

Reply:
Sorry, I don't follow. I was just wondering if we allow also '1m', 'flashcam' and 'nectarcam' in addition to '1M', 'FlashCam' and 'NectarCam'. Technically it is easy to allow, if you simply change the test to 

`if 'flashcam' in telType.lower():`

Do you think that would generate issues?

Reply:
It's easy but it will work only at this point. It won't work when reading from the DB.

I'll try to change the validateTelescopeName to change some key words, like flashcam, 1m, astri, to the right format. It's not very beautiful but it will be quite useful. It's indeed annoying to write FlashCam everytime.
This will fix the problem because, when converting, the name will always be in the right format.

Reply:
Ahh, I didn't realise that it would fail later. I thought that the conversion to the correct name happens anyway before calling the DB. OK, no problem, for now I saw you changed the tests so that they comply with this. I will keep them like that.

Reply:
I just did a PR (and merged it) to add that functionality. It will work fine for all the current existing cases now. We can do something better in the future.